itest/assets_test.go

```diff
@@ -786,7 +786,8 @@ func createAndPayNormalInvoice(t *testing.T, src, rfqPeer, dst *HarnessNode,
 	require.NoError(t, err)
 
 	numUnits, _ := payInvoiceWithAssets(
-		t, src, rfqPeer, invoiceResp, assetID, smallShards,
+		t, src, rfqPeer, invoiceResp.PaymentRequest, assetID, smallShards,
+		fn.None[lnrpc.Payment_PaymentStatus](),
```

Comment:
nit: could add a `defaultPaymentStatus = fn.None[lnrpc.Payment_PaymentStatus]()` at the top of the file to shorten this across this commit.